config.schema.yaml

```diff
@@ -83,7 +83,7 @@ repository_sensor:
       required: false
       default: 30
 activity_sensor:
-  description: "Sensor specific settings, with default github type."
+  description: "This sensor works with git organization / repository registered via register_rog action.  The organizarion configured for repository_sensor should not be registered by the register_org action."
```

Comment:
typo "register_rog" and "organizarion"

Reply:
I think I would prefer if we just flat out say user should use either the respository_sensor or the activity_sensor, not both at the same time.

Reply:
Thanks.  I'll update.